README.md

```diff
@@ -1,178 +1,224 @@
-servers and TimescaleDB. You should make sure your `postgresql.conf`
-settings are tuned, either by using [timescaledb-tune](https://github.com/timescale/timescaledb-tune)
-or doing it manually.
+## Create a hypertable
 
-#### Creating a hypertable
+You create a regular table and then convert it into a hypertable.
 
 ```sql
--- Do not forget to create timescaledb extension
+-- Create timescaledb extension
 CREATE EXTENSION timescaledb;
 
--- We start by creating a regular SQL table
+-- Create a regular SQL table
 CREATE TABLE conditions (
   time        TIMESTAMPTZ       NOT NULL,
   location    TEXT              NOT NULL,
   temperature DOUBLE PRECISION  NULL,
   humidity    DOUBLE PRECISION  NULL
 );
 
--- Then we convert it into a hypertable that is partitioned by time
-SELECT create_hypertable('conditions', 'time');
+-- Convert the table into a hypertable that is partitioned by time
+SELECT create_hypertable('conditions', by_range('time'));
 ```
 
-- [Quick start: Creating hypertables](https://docs.timescale.com/use-timescale/latest/hypertables/create/)
-- [Reference examples](https://tsdb.co/GitHubTimescaleHypertableReference)
+See more:
+
+- [About hypertables](https://docs.timescale.com/use-timescale/latest/hypertables/)
+- [API reference](https://docs.timescale.com/api/latest/hypertable/)
+
+## Insert and query data
+
+Insert and query data in a hypertable via regular SQL commands. For example:
+
+- Insert data into a hypertable named `conditions`:
+
+    ```sql
+    INSERT INTO conditions
+      VALUES
+        (NOW(), 'office', 70.0, 50.0),
+        (NOW(), 'basement', 66.5, 60.0),
+        (NOW(), 'garage', 77.0, 65.2);
+    ```
+
+- Return the number of entries written to the table conditions in the last 12 hours:
+
+    ```sql
+    SELECT COUNT(*) FROM conditions
+      WHERE time > NOW() - INTERVAL '12 hours';
+    ```
+
+See more:
 
-#### Inserting and querying data
+- [Query data](https://docs.timescale.com/use-timescale/latest/query-data/)
+- [Write data](https://docs.timescale.com/use-timescale/latest/write-data/)
 
-Inserting data into the hypertable is done via normal SQL commands:
+## Compress data
+
+You compress your time-series data to reduce its size by more than 90%. This cuts storage costs and keeps your queries operating at lightning speed.
+
+When you enable compression, the data in your hypertable is compressed chunk by chunk. When the chunk is compressed, multiple records are grouped into a single row. The columns of this row hold an array-like structure that stores all the data. This means that instead of using lots of rows to store the data, it stores the same data in a single row. Because a single row takes up less disk space than many rows, it decreases the amount of disk space required, and can also speed up your queries. For example:
+
+- Compress a specific hypertable chunk manually:
+
+    ```sql
+    SELECT compress_chunk( '<chunk_name>');
+    ```
```

Comment:
What about a fully reproducible example... above we demonstrate how to create a hypertable, and now we can demonstrate how turn ON compression in the hypertable and how to compress chunks... Then every user can just copy and paste those code samples and will have it running and working :-)

Reply:
Perfect!